### PR TITLE
feat: added formatBytes function

### PR DESCRIPTION
**Pull request by cristianoap**
This merge request closes https://github.com/elninotech/uppload/issues/402

### CODE REVIEW

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -5,3 +5,12 @@
  */
 export const minifyHTML = (html: string) =>
   html.replace(/\n/g, "").replace(/  /g, "");
+
```
Comment (nicholasgriffintn) [inline src/helpers/utils.ts:8]: Comment for documentation would be good here, such as:

```
/**
 * Returns a formatted version of the bytes
 * @param bytes - The bytes to format
 * @param decimals - The number of decimals to show
 * @returns The formatted string
 */
```

File: src/helpers/utils.ts [hunk 2]
```diff
@@ -5,3 +5,12 @@
...
+export const formatBytes = (bytes: number, decimals: number = 2) => {
+  if (bytes === 0) { return '0 Bytes'; }
+  const k: number = 1024;
```
Comment (nicholasgriffintn) [inline src/helpers/utils.ts:11]: Personally, I'd prefer variables to named, k, dm and i don't mean much and webpack minifies this anyway, so would be better if it was human readable IMO.

**Review by nicholasgriffintn — CHANGES_REQUESTED** [review]

**Review by cristianoap — COMMENTED** [review]
@nicholasgriffintn updated according the comments

**Comment by phlegx** [comment]
@cristianoap nice work!

**Comment by victorlap** [comment]
Thanks!

**Pull request merged**